name:file_error/4
def:file_error(_, {'error',atom()}, #w{}) -> no_return()

